nativeunwind/elfunwindinfo/stackdeltaextraction.go

```diff
@@ -180,6 +180,39 @@ func ExtractELF(elfRef *pfelf.Reference, interval *sdtypes.IntervalData) error {
 	return extractFile(elfFile, elfRef, interval)
 }
 
+// compareStackDelta implements the comparison logic for slices.SortFunc.
+// It must return:
+// -1 if a should come before b (a < b)
+// 0 if a and b are considered equal
+// 1 if a should come after b (a > b)
```

Comment:
Not: strictly the API does not care which number is returned, just that the it's greater/less than zero.